debug.xml

```diff
@@ -458,4 +462,9 @@
     <entry key='oigo.port'>5121</entry>
     <entry key='jpkorjar.port'>5122</entry>
 
+    <!-- UNKNOWN DEVICES CONFIG -->
+    <!-- Automatically adds a new device as soon as it connects for
+    the first time. -->
+    <entry key='unknown.autoAdd'>false</entry>
```

Comment:
I think it's better to rename parameters to something like `database.registerUnknown` because it's related to the database.